test/test_nn.py

```diff
@@ -20537,6 +20537,21 @@ def test_multihead_attn_fast_path_query_and_bias_have_different_dtypes(self, dev
         query = torch.randn(3, 3, 3, dtype=dtype, device=device)
         mha(query, query, query)
 
+    @dtypes(torch.double)
+    @torch.no_grad()
+    def test_multihead_attn_in_proj_bias_none(self, device, dtype):
+        mha = torch.nn.MultiheadAttention(1, 1, bias=True, dtype=dtype, device=device)
```

Comment:
I think you want `bias=False` here to test this case, no?

Reply:
Yeah sorry, forgot to push that